### ISSUE TITLE
collision stop

### ISSUE THREAD

**Issue by MarMar-s** [eclipse-sumo/sumo#14967]
Hello,

I am trying to simulate collisions on the road and have set the collision.stoptime value to 30 seconds to observe queues forming due to the collisions. However, in the simulation, I notice that vehicles mostly go through the collided cars instead of stopping behind them. I have attached a short video of the simulation for your reference. I would be grateful for any assistance you can provide.

Thank you in advance for your reply.

https://github.com/eclipse-sumo/sumo/assets/102868910/37c929eb-a27d-4fdd-97b9-e4447e2f410b

**Comment by m-kro** [comment]
Are the vehicles configured such they would be able to brake in time (e.g. vehicle type attribute tau, decel) before hitting the vehicle in front? If yes, then feel free to provide a small example scenario.

**Comment by MarMar-s** [comment]
Here, to see the effect of limited visibility, I ignore leaders who have a distance of more than 20 meters from their followers using Traci.

**Comment by namdre** [comment]
are you using `setSpeedMode` to override safety features?

**Comment by MarMar-s** [comment]
No, I just use traci.vehicle.setParameter(vehicle, "carFollowModel.ignoreIDs", leader) to ignore leaders when they have a distance of more than 20 meters from their follower.

**Comment by namdre** [comment]
possible reasons for collision:
- removal from ignoreIDs is not working
- even when vehicle are no longer ignored, 20m is not enough to brake safely when there is a high speed difference

You can use sumo-gui to check whether the vehicles start to react and brake at distances below 20m (color-by-acceleration might help)

**Comment by MarMar-s** [comment]
Thank you very much for your time and your reply. I will check it out.

I would like to know if you mean that this may be the reason why other vehicles cross the collided vehicles? I mean that I want the road to be blocked by the collision, but the vehicles pass through it.

**Comment by namdre** [comment]
This does sound as if the "ignoreIDs" feature is still active.

**Comment by m-kro** [comment]
Closing due to lack of feedback
